solutions/instances/main.tf

```diff
@@ -151,7 +152,25 @@ module "kms" {
 # COS
 #######################################################################################################################
 
-module "cos" {
+# workaround for https://github.com/IBM-Cloud/terraform-provider-ibm/issues/4478
+resource "time_sleep" "wait_for_authorization_policy" {
+  depends_on      = [ibm_iam_authorization_policy.policy]
+  count           = var.skip_cos_kms_auth_policy ? 0 : 1
+  create_duration = "30s"
+}
+
```

Comment:
can you leave a code comment to say we are creating the auth policy here instead of in COS module because of this limitation: https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/issues/8